### PR TITLE
Add Opengl documentation (gl4 + gl2.1)

### PR DESCRIPTION
**Pull request by Suraj-Yadav**
If you're adding a new scraper, please ensure that you have:

- [x] Tested the scraper on a local copy of DevDocs
- [x] Ensured that the docs are styled similarly to other docs on DevDocs

- [x] Added these files to the <code>public/icons/*your_scraper_name*/</code> directory:
  - [x] `16.png`: a 16×16 pixel icon for the doc
  - [x] `16@2x.png`: a 32×32 pixel icon for the doc
  - [x] `SOURCE`: A text file containing the URL to the page the image can be found on or the URL of the original image itself

image

### CODE REVIEW

File: lib/docs/scrapers/opengl.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    under a <a href="http://creativecommons.org/licenses/by/4.0/">Creative
+    Commons Attribution 4.0 International License</a>.
+    HTML
+
+    version 'gl2.1' do
+      self.root_path = 'index.html'
+      self.release = 'gl2.1'
+      self.base_url = "https://registry.khronos.org/OpenGL-Refpages/#{self.version}/"
+    end
+    version 'gl4' do
```
Comment (simon04) [inline lib/docs/scrapers/opengl.rb:24]: Typically, we use version numbers w/o any prefixes. Can we drop `gl` for both versions?

File: lib/docs/scrapers/opengl.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      self.root_path = 'index.php'
+      self.release = 'gl4'
+      self.base_url = "https://registry.khronos.org/OpenGL-Refpages/#{self.version}/"
+    end
+
+    def get_latest_version(opts)
+      return 'v1.0.0'
```
Comment (simon04) [inline lib/docs/scrapers/opengl.rb:31]: This should be implemented correctly or omitted. You may want to extract the version from https://registry.khronos.org/OpenGL-Refpages/gl4/html/start.html, for instance.

**Review by simon04 — CHANGES_REQUESTED** [review]
Thank you for your work. Please see and address my comments.

File: lib/docs/scrapers/opengl.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    html_filters.push 'opengl/entries', 'opengl/clean_html'
+
+    # indexflat.php is a copy of index.php
+    options[:skip] = %w(indexflat.php)
+
+    options[:attribution] = <<-HTML
+    Copyright 2017-2021 The Khronos Group, Inc.. This work is licensed
+    under a <a href="http://creativecommons.org/licenses/by/4.0/">Creative
+    Commons Attribution 4.0 International License</a>.
```
[the remark this answers is not in the document]
Comment (Suraj-Yadav) [reply lib/docs/scrapers/opengl.rb]: I have written logic to extract copyright from the page, but I am getting this warning 
`Docs::AttributionFilter is re-parsing the document.`
Unfortunately, I don't think there is any way to do this without parsing the document, as the value changes without any pattern in the URL.

**Comment by Suraj-Yadav** [comment]
@simon04 I have implemented the requested changes.

**Comment by Suraj-Yadav** [comment]
Hi @simon04, Can this be looked into again?

**Review by simon04 — APPROVED** [review]
Thank you!

**Pull request merged**